playbooks/volt-review-app.md

```diff
@@ -0,0 +1,75 @@
+# Volt Review Apps
+
+The authoritative source of information for creating review apps using Hokusai
+can be found here:
+
+https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md
+
+But I've found that a bit noisy and so I'm distilling all that info to a short
+list of commands to run:
+
+```sh
+# create a new config file
+$ hokusai review_app setup 1234
+# update config file with the image tag 1234
+$ sed -i "" "s/staging/1234/g" hokusai/1234.yml
+# commit the config file
+$ git add hokusai/1234.yml
+$ git commit -m "Config for review app 1234"
+# push the image - this takes a while...stop if it doesn't exit cleanly!
+$ hokusai registry push --force --tag 1234
+# create a review app called 1234
+$ hokusai review_app create 1234
+# copy over the env from staging
+$ hokusai review_app env copy 1234 --configmap nginx-config
+# inspect review app and get url
+$ hokusai review_app status 1234
+# or if you're feeling super 1337:
+$ hokusai review_app status 1234 | grep service | tr -s [:blank:] | cut -d ' ' -f 4 | pbcopy
+```
```

Comment:
I was imagining we can contribute the short version back to the canonical hokusai review app doc to help streamline dev's workflow. Is there anything special about Volt's use case?

I'm 👍 to start from this specific example and iterate, so definitely not blocking!